### PR TITLE
Add Implementation of Deep Deterministic Policy Gradient (DDPG) Algorithm

### PR DESCRIPTION
**Pull request by neilkach**
Implemented Deep Deterministic Policy Gradient (DDPG) on Smart Buildings Simulator. Generally followed SAC example and adapted for algorithmic differences as needed.

### CODE REVIEW

**Comment by neilkach** [comment]
Loss graphs from Tensorboard that end up printing "0.00" Actor and Critic loss to stdout after first episode gradient updates: 
image
image

**Comment by bbartling** [comment]
Hi, I follow this repo just curious to ask what setpoints does an RL agent adjust on the HVAC system to optimize? Can you summarize at all? I know HVAC but the RL processes I am still learning...

**Comment by Jgoldfeder** [comment]
Hi @bbartling, we are optimizing water temp setpoint in boilers and air temp setpoint in AHUs
